cluster/proto/api/shard_requests.go

```diff
@@ -94,3 +97,14 @@ type ReplicationDetailsResponse struct {
 	Status        ReplicationDetailsState
 	StatusHistory []ReplicationDetailsState
 }
+
+type ReplicationCancelRequest struct {
+	Version int
+	Uuid    strfmt.UUID
+}
+
+type ReplicationDeleteRequest struct {
+	Version int
+
```

Comment:
nit, remove this newline
```suggestion
```

Reply:
Should this be done for the other structs in this file too?